raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpanGrouperConstants.java

```diff
@@ -21,4 +27,5 @@ public class RawSpanGrouperConstants {
   static final String TRACE_EMIT_PUNCTUATOR_STORE_NAME = "emitter-store";
   static final String TRACE_EMIT_PUNCTUATOR_FREQUENCY_CONFIG_KEY =
       "trace.emit.punctuator.frequency";
+  public static final String PEER_SERVICE_NAME = "peer.service.name";
```

Comment:
this isn't needed. all the attributes enriched by platform goes into enriched attributes map only